tests/Unit_Validator.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import astropy.units
+
+class Unit_Validator():
+
+	def check_unit(self, unit_str):
+		try:
+			return astropy.units.Quantity("1 " + unit_str)
+		except ValueError:
+			return "ValueError: " + unit_str + " is not a valid unit."
+		except:
+			return "UnknownError: " + unit_str
+
+if __name__ == "__main__":
+	Unit_Validator.main()
```

Comment:
How does this work? I would not expect `Unit_Validator` to have a `main` because it is a subclass of `object`. Did you mean to make it a subclass of `unittest.TestCase`?

Reply:
Completely redid this implementation.  The check_unit() function is now in test_utils.py.